### DISCUSSION TITLE
train the learn.py with action=rpm

### DISCUSSION THREAD

**Discussion by hint7** [utiasDSL/gym-pybullet-drones#183] [General]
i've tried [python learn.py # task: single drone hover at z == 1.0] so many times but it seems the rl doesn't work when the action is set to be [rpm]. it is easy to have success when turn action to [1_d_rpm]

**Comment by piratax007** [comment]
It depends on the reward function and the training time (time steps of episodes). Without more details about how you are doing the training is impossible to guess what is going wrong.
